AzureDb.Passwordless.Core/AzureDb.Passwordless.Core.csproj

```diff
@@ -4,6 +4,6 @@
 		<RepositoryUrl>https://github.com/felipmiguel/AzureDb.Passwordless</RepositoryUrl>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="Azure.Identity" Version="1.8.2" />
+	  <PackageReference Include="Azure.Core" Version="1.25.0" />
```

Comment:
the latest version is [1.31.0](https://www.nuget.org/packages/Azure.Core/1.31.0)